Admin/FormAdmin.php

```diff
@@ -142,6 +146,16 @@ function (Localization $localization) {
                 ->addToolbarActions($formToolbarActions)
                 ->setParent(static::EDIT_FORM_ROUTE)
                 ->getRoute(),
+            $this->routeBuilderFactory->createListRouteBuilder(static::LIST_ROUTE_DATA, '/form_data')
```

Comment:
Would use just `data` instead of `form_data`, since `form` is already part of the URL